src/util/member.ts

```diff
@@ -0,0 +1,5 @@
+import type { GuildMember } from 'discord.js';
+
+export const memberHasRoles = (member: GuildMember, ...roles: string[]): boolean => {
```

Comment:
```suggestion
export const hasMemberRoles = (member: GuildMember, ...roles: string[]): boolean => {
```
best practice is to start with verb. this change here is also changes the meaning. "has member roles" can be read as a question, and the answer is a boolean return.
"member has roles" is read as a statement saying that the member does has those role.

Reply:
I first had `hasRoles`, but wanted to be more explicit with `memberHasRole`
doesn't `hasMemberRole` imply as a question if whatever's invoked has something called "MemberRoles"? and not that the `member` is the one we're checking for roles?

Reply:
i think a function for checking only for a specific role would be also against best practices. we want to have more general functions that can be stacked.

`hasRoles` is also a good function name, imo. we also have the context of passing the member object to the function, followed by the roles.